### PR TITLE
feat: add code blocks in quickstart

### PR DESCRIPTION
**Pull request by jainpawan21**
1. Added code blocks to in quickstart page
2. Add Frame component for image
3. Add fullscreen value in youtube video iframe

### CODE REVIEW

File: connectors/google_drive/service_account.mdx [hunk 1]
```diff
@@ -36,8 +36,8 @@ If you'd rather use an individuals account + OAuth to access Google Drive, check
   height="473"
   src="https://www.youtube.com/embed/Z5R9HUCVVAE?si=GZxteF6rmIHa1BOE"
   title="YouTube video player"
-  frameborder="0"
-  allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+  frameBorder="0"
```
Comment (jainpawan21) [inline connectors/google_drive/service_account.mdx:39]: Changed `frameborder` to `frameBorder` to resolve this error

File: connectors/google_drive/service_account.mdx [hunk 2]
```diff
@@ -36,8 +36,8 @@ If you'd rather use an individuals account + OAuth to access Google Drive, check
...
+  allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share; fullscreen;"
```
Comment (jainpawan21) [inline connectors/google_drive/service_account.mdx:40]: With fullscreen value videos can be watched in full screen mode

File: quickstart.mdx [hunk 3]
```diff
@@ -10,14 +10,26 @@ description: "How to deploy Danswer on your local machine"
 
 ## Setup
 **This quickstart guide covers setting up Danswer for local execution**
+
     1. Clone the [Danswer](https://github.com/danswer-ai/danswer) repo:
-    `git clone https://github.com/danswer-ai/danswer.git`
+    ```bash
+    git clone https://github.com/danswer-ai/danswer.git
```
Comment (jainpawan21) [inline quickstart.mdx:16]: It is easy to copy code blocks 🙂

**Comment by yuhongsun96** [comment]
Thanks! Good changes!

**Pull request merged**